rdma-core.sh

```diff
@@ -0,0 +1,9 @@
+package: rdma-core
```

Comment:
This is fine. It would probably make sense to use the prefer_system mechanism and keep track of the version so that we get a rebuild if it changes.